mitosheet/mitosheet/code_chunks/step_performers/dataframe_steps/dataframe_delete_code_chunk.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+from typing import List
+
+from mitosheet.code_chunks.code_chunk import CodeChunk
+
+
+class DataframeDeleteCodeChunk(CodeChunk):
+
+    def get_display_name(self) -> str:
+        return 'Deleted Dataframe'
+    
+    def get_description_comment(self) -> str:
+        sheet_index = self.get_param('sheet_index')
+        df_name = self.prev_state.df_names[sheet_index]
+        return f'Deleted {df_name}'
+
+    def get_code(self) -> List[str]:
+        old_dataframe_name = self.get_param('old_dataframe_name')
+
+        return [f'del {old_dataframe_name}']
```

Comment:
It would be nice to have a combine left function as well as a combine right function - as this would make the expression of the `DataframeDeleteCodeChunk` much more natural - we could specify it in a single place, rather than everywhere!

But does this complicate things and make things too hard to reason about? First you do a right optimize path? Then you do a left optimize pass.